charts/stable/qbittorrent/values.yaml

```diff
@@ -39,7 +39,8 @@ service:
 workload:
   qbitportforward:
     enabled: true
-    type: Deployment
+    type: CronJob
+    schedule: "5 * * * *"
```

Comment:
Probably this needs to be exposed to UI as well
Eg set it to {{ .Values.qbitportforward.schedule }} and add ui field and also under qbitportforward in values.yaml

Reply:
I figured just run it every 5 minutes like upstream script should if it did not try and use root.

Reply:
Then you mean */5 ?

Reply:
Whoops

Reply:
Is what I have set not also using 5 minutes?

Reply:
Ahh